### ISSUE TITLE
volatile variables for evaluation

### ISSUE THREAD

**Issue by martinjankowiak** [uber/pyro#136]
In certain places (e.g. `kl_qp.eval_objective()`) we could have pyro automatically set input `Variable`s to have `volatile=True`. This can result in a speed-up. Or should this be entirely on the user? Thoughts?

**Comment by fritzo** [comment]
This is now possible via `with torch.no_grad()`, which is the default behavior of `svi.loss()` as of #1060.
